arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/TransactionStack.java

```diff
@@ -31,6 +33,7 @@ public class TransactionStack implements StackedContainer {
       new ArrayList<>(200); // TODO: write the allocated memory from .toml file
   private int currentAbsNumber;
   private int relativeTransactionNumber;
+  @Setter @Getter public TxInitializationSection initializationSection;
```

Comment:
The initialization section is required for undoing the initial value transfer during the finalization phase. As such it is only required when the transaction fails.